contents/docs/_snippets/groups.mdx

```diff
@@ -0,0 +1,328 @@
+> **Note: ** This feature is only available on our paid plan. See our [pricing page](/pricing) for more.
+
+**Groups** are a powerful feature in PostHog that aggregate events based on entities, such as organizations or companies. They enable you to analyze trends, insights, and dashboards at an entity-level, as opposed to a user-level.```
+
+To clarify what we mean, let's look at a few examples:
+
+1. For B2B SaaS apps, you can create a **company** group type. This enables you to aggregate events at a company-level, and calculate metrics such as `number of daily active companies`, `company churn rate`, or `how many companies have adopted a new feature`.```
+
+2. For collaborative, project-based apps like Notion, Jira, or Figma, you can create a **project** group type. This enables you to calculate metrics like `project pageviews`, `users per project`, and `project engagement`.
+
+3. For a communication app like Slack, you can create a **channel** group type. This enables you to measure metrics like `average number of messages per channel`, `number of monthly active channels`, or `total number of channel participants`.
+
+4. For a social media app like Twitter, you can create a **post** group type. This enables you to measure `average number of replies per post`, or `total count of unique posters per month`.
+
+> Groups are often confused with [cohorts](/docs/data/cohorts), but they each serve different purposes:
+>
+> - Groups aggregate events, and do not necessarily have to be connected to a user.
+> - Cohorts represent a specific set of users – e.g., a list of users that all belong to the same company.
+> 
+> Groups require additional code in your app to set up, while cohorts are created in PostHog and don't require additional code. This makes cohorts easier to use and quicker to get started. If your only goal is to create a list of **users** with something in common, we recommend cohorts instead of groups.```
```

Comment:
```suggestion
## The difference between groups and cohorts

Groups are often confused with [cohorts](/docs/data/cohorts), but they each serve different purposes:

- Groups aggregate events, and do not necessarily have to be connected to a user.
- Cohorts represent a specific set of users – e.g., a list of users that all belong to the same company.

Groups require additional code in your app to set up, while cohorts are created in PostHog and don't require additional code. This makes cohorts easier to use and quicker to get started. If your only goal is to create a list of **users** with something in common, we recommend cohorts instead of groups.```
```

Reply:
Better have this with a heading than a boxout. It's quite important and makes the section more searchable.